src/MagicOnion.Server/Hubs/Group.ImmutableArray.cs

```diff
@@ -111,17 +111,20 @@ public ValueTask<bool> RemoveAsync(ServiceContext context)
         {
             lock (gate)
             {
-                members = members.Remove(context);
-                if (inmemoryStorage != null)
+                if (this.members.Contains(context))
```

Comment:
Thank you! we have reproduced and confirmed the problem.
It seems that if we skip the process if it is empty, we don't need to check if there is an instance or not, what do you think?

```suggestion
                if (!members.IsEmpty)
```

Reply:
Thank you!
I confirmed the operation and there was no problem, so I fixed it.